src/perf/bin/drvmain.cpp

```diff
@@ -107,6 +107,12 @@ void __cdecl operator delete[] (_In_opt_ void* Mem) {
     }
 }
 
+void __cdecl operator delete[] (_In_opt_ void* Mem, _In_opt_ size_t) {
```

Comment:
Why is this just now needed?

Reply:
Because the compiler used this overload since its not trivially destructible. Its allowed to pick whichever one.